src/OmniSharp.Host/Services/AssemblyLoader.cs

```diff
@@ -90,5 +96,10 @@ public Assembly LoadFrom(string assemblyPath, bool dontLockAssemblyOnDisk = fals
             _logger.LogTrace($"Assembly loaded from path: {assemblyPath}");
             return assembly;
         }
+
+        public Assembly LoadFromPath(string fullPath)
```

Comment:
why do we need to add this, if it already exists as `public Assembly LoadFrom(string assemblyPath, bool dontLockAssemblyOnDisk = false)`? I would suggest to not proliferate these methods if they just call each other